GlotPress/GlotPress-WP
# Local: Add a Locally deploy translations button

## What?
Add the ability to locally deploy translatiions so that they are picked up from PHP.

## Why?
The updated translations were only visible using Inline Translations where the JavaScript would replace them back in (potentially using the wrong original (see #1602)). After locally deploying, PHP will load them.

## How?
We write the PO and MO file in the expected locations.

## Todo in an upcoming PR
This only writes PO files, we don't write JS files yet.

## Testing Instructions
1. Turn off inline translation
2. Go to the Local GlotPress page
3. Press Locally deploy translations
4. Load a page where you can see the new translations loaded from PHP.

## Screenshots or screencast 
Here you can see the local translations being deployed (inline translations are off). Note how fetching translations from translate.wordpress.org will set them back to the WordPress.org state.
![local-translation-deploy](https://user-images.githubusercontent.com/203408/230605688-03a0d708-a219-4096-8589-87b7695a7d11.gif)

This shows how changing a translation where Inline translation takes the wrong one can be changed by locally deploying translations:
![local-translation-deploy-multiple-originals](https://user-images.githubusercontent.com/203408/230606523-8aadc84b-2649-4921-a02d-03e0c327994a.gif)
